package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "superagent",
   "description": "elegant & feature rich browser / node HTTP with a fluent API",
-  "version": "5.2.2",
+  "version": "5.2.3",
```

Comment:
Please do not version bump package.json.  If you remove this commit I will merge it.

Reply:
Thanks @niftylettuce !